pages/common/rustup.md

```diff
@@ -0,0 +1,24 @@
+# rustup
+
+> Rust toolchain installer.
+> Install, manage, and update Rust toolchains.
+
+- Install the nightly toolchain for your system:
+
+`rustup install nightly`
+
+- Switch the default toolchain to nightly so that the cargo and rustc commands will use it:
+
+`rustup default nightly`
+
+- Update all toolchains:
+
+`rustup update`
+
+- List installed toolchains:
+
+`rustup show`
+
+- Run cargo build with a certain toolchain:
+
+`rustup run {{toolchain name}} cargo build`
```

Comment:
We usually prefer `snake_case`. Could you change this to be `toolchain_name` plese?

Reply:
Sure